include/z64player.h

```diff
@@ -750,7 +750,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE2_11 (1 << 11)
 #define PLAYER_STATE2_12 (1 << 12)
 #define PLAYER_STATE2_LOCK_ON_WITH_SWITCH (1 << 13) // Actor lock-on is active, specifically with Switch Targeting. Hold Targeting checks the state of the Z button instead of this flag.
-#define PLAYER_STATE2_14 (1 << 14)
+#define PLAYER_STATE2_FROZEN (1 << 14)
```

Comment:
Currently "frozen" is used all over the codebase to mean "halted", so we need to be more specific here, like `FROZEN_IN_ICE`.
Also `// Currently frozen in ice. Can break free by spinning the stick or mashing A+B.`